tests/docker/perforce/Dockerfile

```diff
@@ -6,11 +6,11 @@ ENV DEBIAN_FRONTEND noninteractive
 RUN apt-get update && apt-get install -y wget gnupg2
 
 # Get perforce packages
-RUN wget -q http://package.perforce.com/perforce.pubkey -O - | apt-key add - && \
-    echo "deb http://package.perforce.com/apt/ubuntu bionic release" > /etc/apt/sources.list.d/perforce.list && \
-    apt update
+#RUN wget -q http://package.perforce.com/perforce.pubkey -O - | apt-key add - && \
```

Comment:
What is the purpose of these changes?

Reply:
oops, these were not supposed to be committed